Makefile

```
@@ -105,7 +105,9 @@ endef
 
 # TODO: Define more specific shells.
 TARGET := default
-SHELL := ./nix/scripts/shell.sh
+ifneq ($(detected_OS),Windows)
+	SHELL := ./nix/scripts/shell.sh
```

The indent is unnecessary. This is not a target.
```suggestion
SHELL := ./nix/scripts/shell.sh
```